sci-libs/rocBLAS/rocBLAS-5.1.3.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DOCS_BUILDER="doxygen"
+DOCS_DIR="docs"
+DOCS_DEPEND="media-gfx/graphviz"
+ROCM_VERSION=${PV}
+inherit cmake docs edo multiprocessing prefix rocm
+
+DESCRIPTION="AMD's library for BLAS on ROCm"
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/rocBLAS"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-${PV}.tar.gz -> rocm-${P}.tar.gz
+	https://media.githubusercontent.com/media/littlewu2508/littlewu2508.github.io/main/gentoo-distfiles/${PN}-5.0.2-Tensile-asm_full-navi22.tar.gz"
+S="${WORKDIR}/${PN}-rocm-${PV}"
+
+LICENSE="BSD"
+KEYWORDS="~amd64"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="benchmark test"
+REQUIRED_USE="${ROCM_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	dev-util/rocm-cmake:${SLOT}
+	dev-util/Tensile:${SLOT}
+"
+
+DEPEND="
+	dev-util/hip:${SLOT}
```

Comment:
Compared to 5.0.2, why do you lock the hip version?

```patch
 DEPEND="                  
-       dev-util/hip:=               
+       dev-util/hip:${SLOT}                                                                              
        dev-libs/msgpack                                                                                                                                                                                            
        test? (                                    
                virtual/blas                      
```

Reply:
I made a mistake here. Now rocBLAS is free of hip version.